call.js

```diff
@@ -0,0 +1,153 @@
+// const fun1 = (callback1) =>{
+//     return callback1()
+// }
+// const fun2 = () => {
+//     return "hello"
+// }
+// console.log(fun1(fun2))
+
+// const boo1 = (boolean, callback) => {
+//     if (boolean === true){
+//         return callback()
+//     } else{
+//         return "ignoring the callback"
+//     }
+// } 
+// const callback = () =>{
+//     return "true"
+// }
+// console.log(boo1(true, callback))
+
+//Question 3
+// const called = (callback, num) => {
+//        let str = callback
```

Comment:
why would we set a variable called string to the value of a function? Dont do this. 
Need to later invoke function